docs/source/tutorial/tutorial-authentication.mdx

```diff
@@ -1,38 +1,36 @@
 ---
-title: Authentication
+title: 7. Enable authentication
 ---
 
 import LoginUISetupPanel from "./components/login_ui_setup_panel.mdx"
 
-In this section, you'll log in to the server and get a token you can use to make identified requests. 
+In this section, you'll add the ability to log in to your server and obtain a token that your client can use to make identified requests.
 
-> **Note**: The way you log in with this particular server may be very different from the way you log in with your own server - login is often handled by _middleware_, or a layer totally separate from GraphQL.
+> **Note**: The way you log in to this particular server might differ from the way you log in with your own server. Login is often handled by _middleware_, or a layer totally separate from GraphQL.
 > 
-> However you get the token from your server, you'll be sending it back to the server the same way in the next part of this tutorial, and can still learn some valuable things from this section of the tutorial.
+> Regardless of how you obtain the token from your server, you'll send it back to the server the same way in the next part of this tutorial.
 
-The first thing you'll do is create a **mutation** to log yourself in.
+## Create a login mutation
 
-## Creating a login mutation
+A **mutation** is an operation that changes state on your server. In this case, the mutation changes back-end state by creating a login session for your client.
```

Comment:
maybe `by creating a session tied to a particular user of your client` - `a login session for your client` doesn't really mean anything